Replicator/Replicator.cc

```diff
@@ -609,6 +609,15 @@ namespace litecore::repl {
     void Replicator::_onConnect() {
         logInfo("Connected!");
 
+        // onConnect may come after the replicator is asked to stop. This situation is
+        // guarded against in WebSocketImpl::onConnect. However, LoopbackWebSocket does
+        // not go throuth WebSocketImpl. We catch the case here.
+        if ( _connectionState != Connection::kConnecting ) {
+            logInfo("Method=_onConnect connectionState=%d Replicator is not in Connecting state, ignoring _onConnect",
+                    _connectionState);
+            return;
+        }
```

Comment:
What do you think about only log "Connected"  after this if as it's actually not connected? When having "Method=_onConnect", it feels like a debugging log message. How about something like this?

```
logInfo("Replicator not in connecting state (connectionState=%d); ignoring onConnect.", _connectionState);
```

Reply:
super